docs/getting-started/windows-manual.md

```diff
@@ -87,6 +88,31 @@ Start-Service splunk-otel-collector
 
 The collector logs and errors can be viewed in the Windows Event Viewer.
 
+### Chocolatey installation
+
+#### Package Parameters
+The following package parameters are available:
+
+- `/SPLUNK_ACCESS_TOKEN`: The splunk access token (org token) used to send metric data to OTEL. If not specified default is to `1234`
+- `/SPLUNK_REALM`: The parameter will be saved to the `\ProgramData\Splunk\OpenTelemetry Collector\SPLUNK_REALM` file. If not specified default is ("us0").
+- `/SPLUNK_INGEST_URL:`: URL of the Splunk ingest  (e.g. `https://ingest.YOUR_SPLUNK_REALM.signalfx.com`). Default value is `https://ingest.us0.signalfx.com`.
```

Comment:
`s/YOUR_SPLUNK_REALM/SPLUNK_REALM` throughout?

Reply:
changed `YOUR_SPLUNK_REALM` to `SPLUNK_REALM` throughout.

Reply:
I'm not sure you've pushed all these changes (still see some remaining)